python/src/nnabla/experimental/distributions/multivariate_normal.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2017 Sony Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+import nnabla.functions as F
+
+from .distribution import Distribution
+
+
+class MultivariateNormal(Distribution):
+    """Multivariate normal distribution.
+
+    Multivariate normal distribution defined as follows:
+
+    .. math::
+
+        p(x | \mu, \Sigma) = \frac{1}{\sqrt{(2 \pi)^k \det(\Sigma)}}
+            \exp(-\frac{1}{2}(x - \mu)^T \Sigma^-1 (x - \mu))
+
+    where :math:`k` is a rank of `\Sigma`.
+
+    Args:
+        loc (~nnabla.Variable): N-D array of :math:`\mu` in definition.
+        scale (~nnabla.Variable): N-D array of diagonal entries of :math:`L`
+            such that covariance matrix :math:`\Sigma = L L^T`.
```

Comment:
I expect it takes nn.Variable because equations in all methods of this class are differentiable. And, since the purpose of these classes is to easily build differentiable distributions, I don't think it is necessary to take numpy arrays, however it will be good for use. Then, I'll make numpy arrays acceptable in those classes.